checks/pinned_dependencies_test.go

```diff
@@ -70,12 +70,103 @@ func TestGithubWorkflowPinning(t *testing.T) {
 			filename: "./testdata/workflow-not-pinned.yaml",
 			expected: scut.TestReturn{
 				Error:         nil,
-				Score:         checker.MinResultScore,
+				Score:         checker.MaxResultScore - 2,
 				NumberOfWarn:  1,
+				NumberOfInfo:  1,
+				NumberOfDebug: 0,
+			},
+		},
+	}
+	for _, tt := range tests {
+		tt := tt // Re-initializing variable so it is not changed while executing the closure below
+		t.Run(tt.name, func(t *testing.T) {
+			t.Parallel()
+			var content []byte
+			var err error
+			if tt.filename == "" {
```

Comment:
If the filename is always non-empty, remove this `if tt.filename == ""`

Reply:
@laurentsimon PR updated but there is a build failure that is not related to test or code compilation. Tried replicating it locally using `make all` but no luck.

Reply:
Ok figure out the issue. All fixed and build is green now.

@laurentsimon was think there are 2 `TODOS` in the code https://github.com/ossf/scorecard/pull/906/files#diff-9d1287edaef110f125b1ebb8915a0475bdbb49cd780287ecbad213f3b4d99db9R236 and https://github.com/ossf/scorecard/pull/906/files#diff-9d1287edaef110f125b1ebb8915a0475bdbb49cd780287ecbad213f3b4d99db9R245 that will need to be done and keen to work on it.

That will need separate ticket and bit of pointing in the right direction :)

Reply:
awesome. Thanks for the great PR.